composer.json

```diff
@@ -11,7 +11,7 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
+        "php": "^7.2 || ^8.0",
```

Comment:
Blocked by https://github.com/friends-of-phpspec/phpspec-code-coverage/pull/39 and https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4702.

Reply:
Change back to `">=7.2"`, since this already includes support for PHP 8.